### ISSUE TITLE
Travis error on pyomo.pysp.tests.smps.test_pysp2smps test

### ISSUE THREAD

**Issue by gseastream** [Pyomo/pyomo#199]
In the GAMS solver pull request, this error occurs every other time travis runs the tests for Python 2.7 "parallel". It doesn't seem to have anything to do with the GAMS solver.

Build: https://travis-ci.org/Pyomo/pyomo/builds/262709091
Job: https://travis-ci.org/Pyomo/pyomo/jobs/262709096

```
test_scenarios_MPS_1server (pyomo.pysp.tests.smps.test_pysp2smps.TestPySP2SMPS_Pyro_HandshakeAtStartup_MultipleWorkers_piecewise_alt) ... 

No output has been received in the last 10m0s, this potentially indicates a stalled build or something wrong with the build itself.
Check the details on how to adjust your build configuration on: https://docs.travis-ci.com/user/common-build-problems/#Build-times-out-because-no-output-was-received

The build has been terminated
```

**Comment by jsiirola** [comment]
Every other build?  @ghackebeil: does this look like a problem with the Pyro nameserver/dispatcher not shutting down correctly, or else multiple nameservers/dispatchers running on the same machine (in different builds?)?

**Comment by ghackebeil** [comment]
I thought this went away when we stopped testing with Pyro3 on Travis (which is ancient at this point). If you don't mind sticking this out, we are a day or two away from merging the PySP branch into master, where this should be fixed (if it's not, I'll take a closer look). In the short term, let's just restart the same Travis job until that failure doesn't pop up.

I don't think it's related to multiple nameservers, etc because the tests use unique  ports that they check for availability first (ignoring unlikely race conditions). I wish there was some way to see the output files for that test though.

**Comment by gseastream** [comment]
To restart the travis build, it looks like someone with write access on the repo needs to do:

> On the build's detail screen, there is a button ↻ with the tooltip "Restart Build".

From https://stackoverflow.com/questions/17606874/trigger-a-travis-ci-rebuild-without-pushing-a-commit

**Comment by qtothec** [comment]
I went ahead and opted for option 2: close and reopen pull request.

**Comment by gseastream** [comment]
Yup, test passes now.

**Comment by qtothec** [comment]
@gseastream: is this issue resolved now, then?

**Comment by gseastream** [comment]
I haven't messed around with travis in a while, but I don't remember there actually being a resolution to this problem. There may been updates since this issue that fixed it, but the way we got around it before was just to rerun the travis tests. So I'm not really sure if the problem still exists, but other than that I haven't personally experienced it in a while given I haven't been paying much attention to travis.

**Comment by ghackebeil** [comment]
fixed as of 5cedbff35c217b85e085431d2f89e93497568e9d and 9930927f05ef2c4257942b39866e9d2222d6369b on master
